packages/engine-ui/src/utils/mapUtils.js

```diff
@@ -166,14 +184,15 @@ export const toGeoJsonLayer = (id, data) =>
     lineWidthScale: 1,
     lineWidthMinPixels: 2,
     getFillColor: (d) => hexToRGBA(d.properties.color, 255),
-    highlightColor: [104, 211, 245, 255],
+    highlightColor: [19, 197, 123, 255],
     autoHighlight: true,
-    getLineColor: (d) => hexToRGBA(d.properties.color, 100),
+    getLineColor: (d) => hexToRGBA(d.properties.color, 255),
     getRadius: (d) => d.properties.size || 300,
     getLineWidth: 5,
     getElevation: 30,
     pointRadiusScale: 1,
     pointRadiusMaxPixels: 10,
+    onClick: (event) => callback(event),
```

Comment:
You really don't have to change this as it still does the same thing, but.. since this function onClick exposes the event parameter, you dont need an anonymous function to just pass it along, you could just do this:
```suggestion
    onClick: callback,
```